src/librbd/io/AioCompletion.cc

```diff
@@ -118,10 +118,6 @@ void AioCompletion::complete() {
     image_dispatcher_ctx->complete(rval);
   }
```

Comment:
@dillaman Don't you expect any issue with that now `image_dispatcher_ctx->complete` may be called after `async_op` is finished, i.e. theoretically after the image context may be destroyed?

Reply:
Thanks -- I moved it.